packages/rum-recorder/src/boot/recorder.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { createNewEvent, HttpRequest, isIE } from '@datadog/browser-core'
+import { LifeCycle, LifeCycleEventType } from '@datadog/browser-rum-core'
+
+import { setup, TestSetupBuilder } from '../../../rum-core/test/specHelper'
+
+import { startRecording } from './recorder'
+
+describe('startRecording', () => {
+  let setupBuilder: TestSetupBuilder
+  let sessionId: string | undefined
+
+  beforeEach(() => {
+    if (isIE()) {
+      pending('IE not supported')
+    }
+    sessionId = 'session-id'
+    setupBuilder = setup()
+      .withParentContexts({
+        findView() {
+          return {
+            session: {
+              id: sessionId,
+            },
+            view: {
+              id: 'view-id',
+              referrer: '',
+              url: 'http://example.org',
+            },
+          }
+        },
+      })
+      .beforeBuild(({ lifeCycle, applicationId, configuration, parentContexts, session }) => {
+        return startRecording(lifeCycle, applicationId, configuration, session, parentContexts)
+      })
+  })
+
+  afterEach(() => {
+    setupBuilder.cleanup()
+  })
+
+  it('starts recording', (done) => {
```

Comment:
`startRecording: starts recording` is not very explicit, what system output are we testing here?